### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by v1r3n**
* Separate startup script to start redis and dynomite

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -71,16 +72,9 @@ EXPOSE 22222
 RUN echo 'Exposing client port for Dynomite 8102'
 EXPOSE 8102
```
Comment (ipapapa) [inline docker/Dockerfile]: We don't really need to `EXPOSE 22122` but shall we add it for debugging purposes?

File: docker/scripts/startup.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+#Start redis server on 22122
+redis-server --port 22122 &
+
+src/dynomite --conf-file=conf/redis_single.yml -v11
```
Comment (ipapapa) [inline docker/scripts/startup.sh]: any reason why just don't let it start with default YAML. Any specific reason on selecting `redis_single.yml`?

Comment (v1r3n) [inline docker/scripts/startup.sh]: I used the same command from the existing one.  Let me try with the defaults.

File: docker/Dockerfile (hunk 1 above)
Comment (v1r3n) [inline docker/Dockerfile]: 22122 is already added.

**Pull request merged**